Kconfig

```diff
@@ -29,6 +29,7 @@ config DEFAULT_SMALL
 
 config DEFAULT_TASK_STACKSIZE
 	int "The default stack size for tasks"
+	default 8192 if ARCH_SIM
 	default 2048
```

Comment:
Should we need change to:
default 2048 if !ARCH_SIM

Reply:
why?
i believe kconfig interprets this as "otherwise, 2048"